woocommerce/woocommerce-ios
# Blaze: Connect Blaze Ad Destination with Blaze Campaign Form

Part of #11609 


## Description


This PR enables data passing between Blaze Ad destination screen and Blaze campaign form.

Unit tests are to be added separately in a new PR.

## Testing instructions


1. Start with a Blaze eligible site,
2. Go to a Product, then tap "Promote with Blaze",
3. On the campaign form screen, scroll down to "Ad destination" option and tap it,
4. On the Ad Destination screen, ensure that the "Product URL" is set as Destination URL as the default,
5. Make changes (change destination URL, add URL parameters), then tap Save.
6. Ensure you're returned to campaign form screen.
7. Tap "Ad destination" again.
8. On the Ad Destination screen, ensure that the changes from step 5 are preserved and shown.
